src/js/bootstrap-slider.js

```diff
@@ -1342,6 +1360,27 @@
 
 				return false;
 			},
+			_touchmove: function(ev) {
+				if (ev.changedTouches === undefined) {
+					return;
+				}
+
+				var touch = ev.changedTouches[0];
+
+				var xDiff = touch.pageX - this.touchX;
+				var yDiff = touch.pageY - this.touchY;
+
+				if (!this._state.inDrag) {
+					// Vertical Slider
+					if (this.options.orientation === 'vertical' && (xDiff <= 5 && xDiff >= -5) && (yDiff >=15 || yDiff <= -15)) {
+						this._mousedown(ev);
```

Comment:
I believe this touchmove state I added was just to validate that the user has reached a particular threshold. I see there's a bunch of initialization happening on mousedown that if we only call mousemove, won't happen. I was going with the idea that this touchmove just detects to initiate the actual mousedown using https://github.com/seiyria/bootstrap-slider/blob/master/src/js/bootstrap-slider.js#L1241 to work with the touchmove/mousemove event.